Sources/Sentry/include/HybridPublic/SentryDependencyContainer.h

```diff
@@ -108,7 +108,7 @@ SENTRY_NO_INIT
 
 @property (nonatomic, strong, nullable) SentryFileManager *fileManager;
 @property (nonatomic, strong) SentryAppStateManager *appStateManager;
-@property (nonatomic, strong) SentryThreadInspector *threadInspector;
+- (id<SentryThreadInspecting>)threadInspecting;
```

Comment:
`m`: Why don't we keep this as a property? We can do
```suggestion
@property (nonatomic, strong) id<SentryThreadInspecting> threadInspecting;
```